### PR TITLE
about section added

### PR DESCRIPTION
**Pull request by VivekKumarSahni**
about section component is created. Still to be made responsive .

Screenshot (749)
Screenshot (748)

### CODE REVIEW

File: src/components/about_component/About.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  //   <div>
+  //   <Lottie animationData={aboutus_pc} loop={true} autoplay={true} style={style}/>
+  //   <Lottie animationData={aboutus_coin}  loop={true} style={style}  />
+
+  //   </div  >
+  //   <div className="  w-[100%] bg-[url('./bg2.svg')]" >
+  //   <div className="flex w-[70.5625rem] h-[31.75rem] items-center gap-[7.6875rem] shrink-0 px-40 py-4">
+  // hello i am VIvek
+  //   </div>
+  //   </div>
```
Comment (codadept) [inline src/components/about_component/About.jsx]: Remove the comments if not needed.

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -12,11 +13,14 @@ const Home = () => {
   };
 
   return (
+    <>
     <div>
       <h1 className={styles["heading"]}>Tecnoesis 2024</h1>
       <Input value={demoState} onChange={changeDemoState} />
       <Button>Hello</Button>
     </div>
+    <About></About>
```
Comment (codadept) [inline src/pages/Home/Home.jsx]: Change it to auto closing tag `<About/>`

**Review by codadept — CHANGES_REQUESTED** [review]
Changes
- Why there is two style page About.module.css and About2.module.css? Convert them to one single file
- Follow the file naming convention. Change the folder name to `About` from `about_component`
- Move the lottie files from `src/components/About` to `src/assets` 

Questions
- Why have you added a new font in the folder?

File: src/pages/Home/Home.jsx (hunk 2 above)
Comment (VivekKumarSahni) [reply src/pages/Home/Home.jsx]: Doing the changes bhaiya

File: src/components/about_component/About.jsx (hunk 1 above)
Comment (VivekKumarSahni) [reply src/components/about_component/About.jsx]: The font name JetBrainsMono was not there in the public folder . That's why i added it

Comment (codadept) [reply src/components/about_component/About.jsx]: Alright. Just define the font in App.css as did for other fonts and then use it wherever you need.

File: src/App.css [hunk 3]
```diff
@@ -44,3 +44,7 @@
   font-family: PixeloidSans;
   src: url("/fonts/PixeloidSans-mLxMm.ttf");
 }
+@font-face { 
+  font-family: JetBrainsMono; 
+  src: url("../fonts/JetBrainsMono-VariableFont_wght.ttf");
```
Comment (codadept) [inline src/App.css]: Your fonts are in the public directory, to access the content inside the public directory you can start with "/"

E.g. if you have some font in the directory "public/fonts/font1.ttf"

Then you can define the path in CSS file as

```css
@font-face{
    font-family: font1;
    src: url("/fonts/font1.ttf");
}
```

[Ref.](https://vitejs.dev/guide/assets#the-public-directory)

File: src/index.css [hunk 4]
```diff
@@ -41,3 +41,15 @@ h6 {
...
+  font-size:1.1vw;
+} */
+@font-face { 
+  font-family: "Pixeloid"; 
+  src: url("../fonts/PixeloidSansBold-PKnYd.ttf"); 
+}
+@font-face { 
+  font-family: "JetBrainsMono"; 
+  src: url("../fonts/JetBrainsMono-VariableFont_wght.ttf"); 
+}
```
Comment (codadept) [inline src/index.css]: Already defined in App.css file. No need of it here.

**Review by codadept — CHANGES_REQUESTED** [review]
Do these changes, rest are fine.

File: src/index.css (hunk 4 above)
Comment (VivekKumarSahni) [reply src/index.css]: The changes are done bhaiya.

**Review by codadept — APPROVED** [review]

**Pull request merged**